File: WinForms/VB/RichEditMainDemo/Modules/Navigation.vb
Imports DevExpress.XtraBars.Ribbon

Namespace DevExpress.XtraRichEdit.Demos

    Public Partial Class NavigationModule
        Inherits TutorialControl

        Public Sub New()
            InitializeComponent()
            Dim ribbonControl As RibbonControl = richEditControl.CreateRibbon(RichEditToolbarType.File Or RichEditToolbarType.Home Or RichEditToolbarType.Insert Or RichEditToolbarType.References)
            Controls.Add(ribbonControl)
            ribbonControl.SelectedPage = ribbonControl.Pages("References")
            LoadDocument("Navigation.docx")
        End Sub

        Public Overrides ReadOnly Property RichEdit As RichEditControl
            Get
                Return richEditControl
            End Get
        End Property
    End Class
End Namespace

File: WinForms/CS/RichEditMainDemo/Modules/Navigation.cs
using DevExpress.XtraBars.Ribbon;

namespace DevExpress.XtraRichEdit.Demos {
    public partial class NavigationModule : TutorialControl {
        public NavigationModule() {
            InitializeComponent();
            RibbonControl ribbonControl = richEditControl.CreateRibbon(RichEditToolbarType.File | RichEditToolbarType.Home | RichEditToolbarType.Insert | RichEditToolbarType.References);
            Controls.Add(ribbonControl);
            ribbonControl.SelectedPage = ribbonControl.Pages["References"];
            LoadDocument("Navigation.docx");
        }

        public override RichEditControl RichEdit { get { return richEditControl; } }
    }
}
